python/nav/web/seeddb/forms/__init__.py

```diff
@@ -325,12 +325,10 @@ class OrganizationMoveForm(forms.Form):
 class NetboxTypeFilterForm(forms.Form):
     """Form for filtering a netbox type by vendor"""
 
-    vendor = forms.ModelChoiceField(Vendor.objects.order_by('id').all(), required=False)
-
-    def __init__(self, *args, **kwargs):
-        super(NetboxTypeFilterForm, self).__init__(*args, **kwargs)
-        self.helper = get_formhelper()
-        self.helper.layout = get_single_layout('Filter types', 'vendor')
+    vendor = forms.ModelChoiceField(
+        Vendor.objects.order_by('id').all(), required=False, label_suffix=''
+    )
```

Comment:
```suggestion
    )
    vendor.widget.attrs.update({"class": "select"})
```

Reply:
I used it to show how to set classes on the actual input-tag from python, but if it doesn't do any thing we do not need it.